File: Licenta.Server/DataLayer/Dto/AddProjectDto.cs
using Microsoft.AspNetCore.Mvc;

namespace Licenta.Server.DataLayer.Dto
{
    public class AddProjectDto
    {
        public Guid ProjectId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public Guid OwnerId { get; set; }
    }
}

File: Licenta.Frontend/src/app/Models/DTO/AddProjectDto.ts
export class AddProjectDto {
    name: string;
    description: string;
    startDate: Date;
    endDate: Date;
    ownerId: string;

    constructor()
    {
        this.name = '';
        this.description = '';
        this.startDate = new Date();
        this.endDate = new Date();
        this.ownerId = '';
    }
}